configs/default/IFRC-IFLIGHT_F745_AIO.config

```diff
@@ -107,27 +118,56 @@ dma pin A03 0
 
 # feature
 feature -RX_PARALLEL_PWM
+feature -AIRMODE
 feature RX_SERIAL
+feature TELEMETRY
 feature LED_STRIP
 feature OSD
+feature ESC_SENSOR
 
 # serial
-serial 2 64 115200 57600 0 115200
+serial 0 8192 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 5 1024 115200 57600 0 115200
+serial 6 2 115200 57600 0 115200
 
 # led
 led 0 6,11::CTOBVIW:0
 led 1 7,11::CTOBVIW:0
 led 2 8,11::CTOBVIW:0
 led 3 9,11::CTOBVIW:0
 
+# aux
+aux 0 0 0 900 1300 0 0
+
+# vtxtable
+vtxtable bands 6
+vtxtable channels 8
+vtxtable band 1 BOSCAM_A A CUSTOM  5865 5845 5825 5805 5785 5765 5745 5725
+vtxtable band 2 BOSCAM_B B CUSTOM  5733 5752 5771 5790 5809 5828 5847 5866
+vtxtable band 3 BOSCAM_E E CUSTOM  5705 5685 5665 5645 5885 5905 5925 5945
+vtxtable band 4 FATSHARK F CUSTOM  5740 5760 5780 5800 5820 5840 5860 5880
+vtxtable band 5 RACEBAND R CUSTOM  5658 5695 5732 5769 5806 5843 5880 5917
+vtxtable band 6 BAND_D   D CUSTOM  5362 5399 5436 5473 5510 5547 5584 5621
+vtxtable powerlevels 5
+vtxtable powervalues 25 100 200 400 600
+vtxtable powerlabels 25 100 200 400 600
```

Comment:
Unless you are willing to certify that these settings are legal worldwide (and provide indemnity to anybody using this board), then you cannot include the `vtxtable` in the unified target. And clearly these settings **are not legal worldwide.**